ClimsoftVer4/ClimsoftVer4/ucrFormDaily2.vb

```diff
@@ -42,6 +43,8 @@ Public Class ucrFormDaily2
                     fd2Record = New form_daily2
                     bUpdating = False
                 End Try
+                'This is determined by the current user not set from the form
+                fd2Record.signature = frmLogin.txtUsername.Text
```

Comment:
This wasn't being added to the record when it is in the old forms. This needs to be included on all forms.